Sources/App/CRP/JiraService+CRP.swift

```diff
@@ -18,11 +19,20 @@ extension JiraService {
             summary: crpConfig.jiraSummary(release),
             environments: [crpConfig.environment],
             release: release,
+            releaseType: .init(version: release.version),
+            targetDate: targetDate ?? guessTargetDate(),
             changelog: changelog,
             accountablePersonName: accountablePerson
         )
         return CRPIssue(fields: fields)
     }
+
+    private static func guessTargetDate() -> Date {
+        let now = Date()
+        // Estimate time between when the CRP ticket is created and the app is released to the AppStore
+        let estimateOffset = DateComponents(day: 5)
```

Comment:
This is very optimistic :D

Reply:
Actually I put 5 here because Mark agreed that 1 week was a good enough estimate and I put 5 thinking "5 working days in a week"… but I should have put 7 instead, _especially_ since we'll soon cut the release branch on Friday morning and likely create the CRP on Friday, so at least 7 should be closer to the reality 😉 

```suggestion
        let estimateOffset = DateComponents(day: 7)
```